.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ executors:
   test-executor:
     resource_class: medium
     docker:
-      - image: checkclientqualifiesdocker/circleci-image:puppeteer-2263
+      - image: checkclientqualifiesdocker/circleci-image:pupeteer-2265
```

Comment:
@starswan - typo/spelling of 'puppeteer'

Reply:
I know - the branch name is wrong